src/app/onboarding/components/create-new-archive/create-new-archive.component.html

```diff
@@ -1,101 +1,123 @@
-<div>
-  <div *ngIf="screen === 'type'">
-    <h2>Create Your First Archive: Select Type</h2>
-
-    <p>What will your first archive represent? It might be yourself, a family member, an ancestor, your neighborhood association, a small business or a nonprofit organization.</p>
-
-    <p>Not sure what archive option to choose? Start with yourself. You can always create new archives or change the name and type of your archive later.</p>
-
-    <p><strong>What type of archive will this be?</strong> You can always change this later.</p>
-
-    <div class="archive-type-selector">
-      <label>
-        <div [ngClass]="getArchiveTypeClasses('type.archive.person')">
-          <img src="assets/svg/onboarding/archive-person.svg" alt="Person Archive" />
-          <div>
-            <strong class="archive-type-name">Person Archive</strong>
-            <span class="archive-type-description">represents an individual person, such as yourself or someone else</span>
+<!-- format -->
+<div class="new-account-screens-container">
+  <div *ngIf="screen === 'create'">
+    <h2>Welcome! Let’s create your first archive together.</h2>
+    <div class="welcome-message no-invitations">
+      <p>
+        In order to help you on this journey, we want to learn a little more
+        about you to personalize your experience and make archiving easier.
+        Please answer the following questions:
+      </p>
+      <div class="archive-type">
+        <h3>
+          With my first archive, I plan to capture and preserve material about…
+        </h3>
+        <span class="select-option">
+          Select the option that fits you best.
+        </span>
+        <pr-archive-type-select
+          (valueChange)="onValueChange($event)"
+          [selectedValue]="selectedValue"
+        ></pr-archive-type-select>
+      </div>
+      <div class="archive-name-container">
+        <div class="archive-name">
+          <h3>What is your new archive called?</h3>
+          <p>You can always edit your archive's name later.</p>
+          <div class="archive-name-input">
+            <span>The</span>
+            <input
+              [(ngModel)]="name"
+              [placeholder]="'archive subject'"
```

Comment:
Even though the design has "archive subject" in all lowercase, I think we should capitalize it here. (Though ultimately, I think this might be up to @k8lyn6 )

Reply:
@meisekimiu I think that makes sense.